src/app/app-routing.module.ts

```diff
@@ -1,8 +1,20 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LandingPageComponent } from './components/landing-page/landing-page.component';
+import { QuestionApprovalPageComponent } from './components/question-approval-page/question-approval-page.component';
+import { PagenotfoundComponent } from './components/pagenotfound/pagenotfound.component';
+import { ActionsComponent } from './components/actions/actions.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+
+  { path: 'actions', component: ActionsComponent },
+  { path: 'actions/approve', component: QuestionApprovalPageComponent },
+  { path: '', component: LandingPageComponent },
+  { path: 'home', component: LandingPageComponent },
```

Comment:
generally better to put the less specific routes later in the array (angular's docs advise this).